### DISCUSSION TITLE
SSH pass

### DISCUSSION THREAD

**Discussion by brunobergamo** [quantumsheep/sshs#108] [Ideas]
Do you have any idea to integrate with sshpass ? 

Best Regards
Bruno

**Comment by quantumsheep** [comment]
Hello 👋 You can use `--template`.

```bash
sshs --template='sshpass -p yourpassword ssh "{{{name}}}"'
```

**Comment by brunobergamo** [comment]
Thanks a lot !

I noted the instruction to install arch version is set to an old version

Best Regards,


![image](https://github.com/user-attachments/assets/4637d335-3101-4b2d-9e15-d08f990a8483)

**Reply by brunobergamo** [reply]
maybe the right one is [sshs-bin](https://aur.archlinux.org/packages/sshs-bin)

**Reply by quantumsheep** [reply]
`sshs` is unmaintained. `ssh-bin` is the right one to choose.
